module/dkg/emulator_client.go

```diff
@@ -0,0 +1,110 @@
+package dkg
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc"
+
+	"github.com/onflow/cadence"
+	jsoncdc "github.com/onflow/cadence/encoding/json"
+
+	emulator "github.com/onflow/flow-emulator"
+
+	sdk "github.com/onflow/flow-go-sdk"
+)
+
+// EmulatorClient is a wrapper around the emulator to implement the same interface
+// used by the SDK client. Used for testing against the emulator.
+type EmulatorClient struct {
+	blockchain *emulator.Blockchain
+}
```

Comment:
We have the same for the `QCClient` as well right? It would be good to merge `feature/epochs` in here and consolidate these.

Reply:
> We have the same for the `QCClient` as well right? It would be good to merge `feature/epochs` in here and consolidate these.

@jordanschalm I tried merging `feature/epochs` into `feature/dkg`  to see if I could help with this. I am having difficulty getting the appropriate version of the core-contracts module:
https://github.com/onflow/flow-go/blob/512bc5d0d0cea7708092b23aafee9525898dd689/go.mod#L40-L41

The tag `v0.7.2` does not seem to exist (https://github.com/onflow/flow-core-contracts/tags)

Reply:
Hey martin, I think the way Go encodes unnamed versions is by prefixing them with a semver version one patch version greater than the last named semver tag in the history. `v0.7.1` is the latest in the history. We are pinning a commit (https://github.com/onflow/flow-core-contracts/commit/ca3212c272a7) but the `v0.7.2` is prefixed for ordering (or something like that). 

What we should be depending on is the head of the https://github.com/onflow/flow-core-contracts/tree/feature/epochs branch in `flow-core-contracts`. When merging we can pin using the latest commit from that branch. Then we'll tag a version when we merge to master.